en/components/tree.md

```diff
@@ -0,0 +1,313 @@
+---
+title: Angular Tree Component - Ignite UI for Angular
+_description: With Ignite UI for Angular Tree component, you can display hierarchical data in a tree-view structure, customize nodes easily and load data on demand. Try for FREE
+_keywords: angular tree, angular tree component, ignite ui for angular, UI controls, infragistics
```

Comment:
Has @aatanasov2 gone through the title, description and keywords?

Reply:
@kdinev He reviewed the entire topic. Actually, title and description are modified as per his suggestions. In regards to the key words, he didn`t leave any comments so we assumed that they are fine.

Reply:
@kacheshmarova @kdinev meta keywords tags are no longer used by the major search engines. that's why I didn't comment there. I tend not to use meta keywords at all, but the list like the above ( 1-2 generic + 1-2 branded terms) is the best approach.

Reply:
@aatanasov2 I changed the wording in a few places in the article. Could you take a look at the last commit I've done to ensure I haven't messed anything up?

Reply:
@kdinev all good with the changes